src/main/java/florizz/core/Ui.java

```diff
@@ -254,21 +254,12 @@ public void printBackPage() throws FlorizzException{
         } else {
             switch (lastCommand.split(" ")[0]) {
             case ("ALL_FLOWERS"):
-                if (lastPageNo*PAGE_SIZE >= lastShownList.size()){
-                    throw new FlorizzException("There is no next page, type 'back' to go to the previous page");
-                }
                 printAllDictFlowerName(lastPageNo-1);
                 break;
             case ("FILTERED_FLOWERS"):
-                if (lastPageNo*PAGE_SIZE >= lastShownList.size()){
-                    throw new FlorizzException("There is no next page, type 'back' to go to the previous page");
-                }
                 printFilteredFlowers(lastShownList, lastCommand.split(" ")[1],lastPageNo-1);
                 break;
             case ("INFO_FLOWERS"):
-                if (lastPageNo*PAGE_SIZE >= lastShownList.size()){
-                    throw new FlorizzException("There is no next page, type 'back' to go to the previous page");
-                }
```

Comment:
This one was @samuelory lines, so I am not sure if the next page functionality is still there if changed